app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -52,9 +52,17 @@
   <section>
     <h2 class="heading-s">Engineers teach physics training programme</h2>
 
-    <p>This new teacher training programme helps engineers to become physics teachers.</p>
+    <p>This new programme is based on the physics initial teacher training (ITT) course. It has been adapted to help you make the transition from engineering or material sciences to teaching physics. The course is designed to capitalise on your experience and empower you with the skills needed to become a great physics teacher.
+     </p>
+
+    <p>Your training provider may offer part-time teacher training courses and may also allow you to combine courses, for example Engineers teach physics combined with mathematics.
+     </p>
 
-    <p>You’ll be able to find courses starting in the 2023/24 academic year from 17 October 2022.</p>
+    <p>If you’re concerned about how well you know physics as a subject, do not worry - you’ll be able to do a ‘subject knowledge enhancement’ (SKE) course that will provide you with the training and support for any gaps in your knowledge.
```

Comment:
Do we want to link to the SKE page here?

Reply:
Good idea!